src/azul/logging.py

```diff
@@ -169,6 +175,7 @@ def http_body_log_message(body_type: str,
                           body: bytes | bytearray | str | None,
                           *,
                           verbatim: bool = False,
+                          size_only: bool = False,
```

Comment:
```suggestion
                          size_only: bool = False
```

Reply:
I'd rather not add this parameter because its relationship with verbatim is not immediately apparent. Logging the size is cheap so it doesn't need to be conditional. This method is used for ES, AWS and general HTTP requests, too, so I am fine if the size is logged for them as well.